diff --git a/lib/lazystream.js b/lib/lazystream.js
index d9f6170965b3d5ece94b094cee180791c74814a9..c98ba291222657d17b54a167e5bd2d5c5f5982aa 100644
--- a/lib/lazystream.js
+++ b/lib/lazystream.js
@@ -1,5 +1,5 @@
 var util = require('util');
-var PassThrough = require('readable-stream/passthrough');
+var PassThrough = require('readable-stream').PassThrough;
 
 module.exports = {
   Readable: Readable,
